packages/calling-component-bindings/review/beta/calling-component-bindings.api.md

```diff
@@ -190,7 +191,7 @@ export interface CommonCallingHandlers {
     // (undocumented)
     onSetSpokenLanguage: (language: string) => Promise<void>;
     // (undocumented)
-    onStartCall: (participants: (CommunicationUserIdentifier | PhoneNumberIdentifier | UnknownIdentifier)[], options?: StartCallOptions) => void;
+    onStartCall: (participants: (CommunicationUserIdentifier | PhoneNumberIdentifier | UnknownIdentifier | MicrosoftTeamsAppIdentifier)[], options?: StartCallOptions) => void;
```

Comment:
why cant we just make this CommunicationIdentifier and save ourselves from having to consider every type? Is it because we just dont want to support calling a teams user directly?